src/phockup.py

```diff
@@ -128,24 +128,32 @@ def walk_directory(self):
 
         # Walk the directory
         for root, dirnames, files in os.walk(self.input_dir):
-            files.sort()
-            file_paths_to_process = []
-            for filename in files:
-                if filename in ignored_files:
-                    continue
-                file_paths_to_process.append(os.path.join(root, filename))
-            if self.max_concurrency > 1:
-                if not self.process_files(file_paths_to_process):
-                    return
+            logger.info(f"walking directory. current dir: {root}, dirnames: {dirnames}, files: {files}")
```

Comment:
No need to use exact variable names in the logs. Please change to `Walking directory: {root}. Directory names: {dirnames}, files: {files}`. Also isn't `files` going to list a huge list of files for this log if the diretory has a lot of files?

Reply:
Actually i don't really need these lines - they were more for debugging and i was just being lazy. i'll get rid of them